packages/app-accounts/src/Vanity/index.tsx

```diff
@@ -98,6 +98,7 @@ class VanityApp extends React.PureComponent<Props, State> {
               ? t('Stop generation')
               : t('Start generation')
           }
+          submitOnEnter={!isRunning}
```

Comment:
Button doesn't listen to prop changes so I think only the initial value of the prop will be taken into account here